Adding missing phone factor metadata (#95857)
Azure/azure-content: articles/active-directory-b2c/phone-factor-technical-profile.md

| ManualPhoneNumberEntryAllowed| No | Specify whether or not a user is allowed to manually enter a phone number. Possible values: `true`, or `false` (default).|
| setting.authenticationMode | No | The method to validate the phone number. Possible values: `sms`, `phone`, or `mixed` (default).|
| setting.autodial| No| Specify whether the technical profile should auto dial or auto send an SMS. Possible values: `true`, or `false` (default). Auto dial requires the `setting.authenticationMode` metadata be set to `sms`, or `phone`. The input claims collection must have a single phone number. |

> | setting.autosubmit | No | Specifies whether the technical profile should auto submit the one-time password entry form. Passible values: `true` (default), or `false` When auto-submit is turned off, the user will have to click a button to progress the journey. |

Reviewer: ```suggestion
| setting.autosubmit | No | Specifies whether the technical profile should auto submit the one-time password entry form. Passible values are `true` (default), or `false`. When auto-submit is turned off, the user will have to select a button to progress the journey. |
```

> | setting.autosubmit | No | Specifies whether the technical profile should auto submit the one-time password entry form. Possible values are `true` (default), or `false`. When auto-submit is turned off, the user will have to select a button to progress the journey. |

Reviewer: ```suggestion
| setting.autosubmit | No | Specifies whether the technical profile should auto submit the one-time password entry form. Possible values are `true` (default), or `false`. When auto-submit is turned off, the user needs to select a button to progress the journey. |
```

Reviewer: @bolt-io 

One last correction before we merge :)

Author: Many thanks for the corrections!
